easybuild/easyconfigs/s/structure/structure-2.3.4.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild recipy by Christian Meesters (2017)
+# HPC-group University of Mainz (https://hpc.uni-mainz.de)
+
+easyblock = 'MakeCp'
+name = 'structure'
+version = '2.3.4'
+
+homepage = 'https://web.stanford.edu/group/pritchardlab/structure.html'
+description = """The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include infering the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed. It can be applied to most of the commonly-used genetic markers, including SNPS, microsatellites, RFLPs and AFLPs."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://web.stanford.edu/group/pritchardlab/structure_software/release_versions/v%(version)s/']
+sources     = ['structure_kernel_source.tar.gz']
```

Comment:
If no properly versioned source tarball is available, please let EasyBuild rename it on download, and also include a SHA256 checksum:

```python
sources = [{'download_filename': 'structure_kernel_source.tar.gz', 'filename': SOURCE_TAR_GZ}]
checksums = ['...']
```